### PR TITLE
Docker improvements 2

### PR DESCRIPTION
**Pull request by Daniel-Marynicz**

### CODE REVIEW

File: config/postgres/docker-entrypoint-initdb.d/20-create-app-db.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -e
+
+psql -v ON_ERROR_STOP=1 \
+    --dbname "$POSTGRES_DB" <<-EOSQL
+    CREATE DATABASE ${APP_DB} OWNER ${APP_USER};
```
Comment (WojFa) [inline config/postgres/docker-entrypoint-initdb.d/20-create-app-db.sh]: Should a test base be created here?

File: .env.dist [hunk 2]
```diff
@@ -7,9 +7,16 @@ COMPOSE_PROJECT_NAME=ergonode
 POSTGRES_PASSWORD=123
 POSTGRES_USER=postgres
 EXPOSED_POSTGRES_PORT=5432
-POSTGRES_DB=ergonode
+POSTGRES_DB=postgres
```
Comment (WojFa) [inline .env.dist]: We use this environment in this time ?

File: config/postgres/docker-entrypoint-initdb.d/20-create-app-db.sh (hunk 1 above)
Comment (Daniel-Marynicz) [reply config/postgres/docker-entrypoint-initdb.d/20-create-app-db.sh]: Currently when you use command `bin/phing test then phing executes tasks 
`database:drop, database:create`
Even if i add create db for test then `phing` will remove this db and create..

File: .env.dist (hunk 2 above)
Comment (Daniel-Marynicz) [reply .env.dist]: in service postgres this is used.

Comment (Daniel-Marynicz) [reply .env.dist]: but this can be removed , because this values have same values as default PostgreSQL docker image.

**Pull request merged**